internal-import-file/import-document/src/reportimporter/core.py

```diff
@@ -126,6 +126,16 @@ def _download_import_file(self, data: Dict) -> str:
         file_name = os.path.basename(file_fetch)
         file_content = self.helper.api.fetch_opencti_file(file_uri, True)
 
+        """
+        On Windows, the invalid characters are different, so the behavior is not the same as Linux
+        It only happens with free text on local setup running on windows. Never on prod.
+        """
+        os_system = os.name
+
+        # If windows detection, replacement of problematic characters
+        if os_system == "nt":
+            file_name = file_name.replace(":", "_")
```

Comment:
I suggest you use a regular expression to search and replace all invalid characters on windows, which are `\<>:*?"|/`

--> `/[\\/:*?"<>|]/g`